src/glang/graphene_parser.py

```diff
@@ -22,19 +23,21 @@
     StructMemberRedeclaration,
     VoidVariableDeclaration,
 )
-from .parser import lexer_parser as lp
+from glang.parser import lexer_parser as lp
 
 UnresolvedGenericMapping = dict[str, cg.UnresolvedSpecializationItem]
 
 
 class ResolvedPath(str):
+    __slots__ = ()
```

Comment:
Can you explain why we'd want this?

Reply:
https://docs.astral.sh/ruff/rules/no-slots-in-str-subclass/

I'll add that as a comment